tests/e2e/config.go

```diff
@@ -39,6 +39,14 @@ type e2eTestConfig struct {
 		User string `gcfg:"user"`
 		// vCenter password in clear text.
 		Password string `gcfg:"password"`
+		// vmc vCentre cloudadmin username
```

Comment:
I think make check may fail since comment is not ending with a period.

Reply:
make check is successful Chethan, without adding a period to the comment.

marunachalam@marunachala-a01 vsphere-csi-driver % make check
hack/check-format.sh
go: found golang.org/x/tools/cmd/goimports in golang.org/x/tools v0.1.1
hack/check-mdlint.sh
hack/check-shell.sh
hack/check-staticcheck.sh
go: found honnef.co/go/tools/cmd/staticcheck in honnef.co/go/tools v0.2.0
hack/check-vet.sh
hack/check-golangci-lint.sh
golangci/golangci-lint info checking GitHub for tag 'v1.40.1'
golangci/golangci-lint info found version: 1.40.1 for v1.40.1/darwin/amd64
golangci/golangci-lint info installed /Users/marunachalam/go/bin/golangci-lint
INFO [config_reader] Config search paths: [./ /Users/marunachalam/Desktop/CNS-CSI/vmc-create-gc/vsphere-csi-driver /Users/marunachalam/Desktop/CNS-CSI/vmc-create-gc /Users/marunachalam/Desktop/CNS-CSI /Users/marunachalam/Desktop /Users/marunachalam /Users /] 
INFO [config_reader] Used config file .golangci.yml 
INFO [lintersdb] Active 12 linters: [deadcode errcheck gosimple govet ineffassign lll misspell staticcheck structcheck typecheck unused varcheck] 
INFO [loader] Go packages loading at mode 575 (deps|compiled_files|exports_file|files|imports|name|types_sizes) took 2.450440296s 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 100.225583ms 
INFO [linters context/goanalysis] analyzers took 38.713983283s with top 10 stages: buildir: 31.313500293s, nilness: 931.742753ms, misspell: 581.266692ms, S1038: 476.063527ms, fact_purity: 389.971346ms, SA1012: 287.271346ms, lll: 278.211121ms, directives: 229.173913ms, unused: 219.073562ms, S1039: 207.537419ms 
INFO [runner] Issues before processing: 110, after processing: 0 
INFO [runner] Processors filtering stat (out/in): cgo: 110/110, skip_dirs: 110/110, identifier_marker: 21/21, skip_files: 110/110, autogenerated_exclude: 21/110, filename_unadjuster: 110/110, exclude: 21/21, exclude-rules: 1/21, nolint: 0/1, path_prettifier: 110/110 
INFO [runner] processing took 14.56548ms with stages: nolint: 10.659278ms, autogenerated_exclude: 1.978694ms, path_prettifier: 1.124ms, identifier_marker: 407.971µs, skip_dirs: 200.54µs, exclude-rules: 158.726µs, cgo: 16.507µs, filename_unadjuster: 13.007µs, max_same_issues: 1.293µs, max_from_linter: 918ns, uniq_by_line: 799ns, skip_files: 719ns, exclude: 505ns, severity-rules: 500ns, diff: 447ns, source_code: 431ns, max_per_file_from_linter: 330ns, sort_results: 300ns, path_shortener: 284ns, path_prefixer: 231ns 
INFO [runner] linters took 14.570232636s with stages: goanalysis_metalinter: 14.55553367s 
INFO File cache stats: 58 entries of total size 1.4MiB 
INFO Memory: 168 samples, avg is 547.6MB, max is 1288.1MB 
INFO Execution took 17.139067178s